Solutions/Azure Activity/Analytic Rules/SubscriptionMigration.yaml

```diff
@@ -0,0 +1,59 @@
+id: 48c026d8-7f36-4a95-9568-6f1420d66e37
+kind: Scheduled
+name: Subscription moved to another tenant
+description: |
+  Detects when a subscription is moved to another tenant. This may indicate a subscription hijacking event.
+severity: Low
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+queryPeriod: 20m
+queryFrequency: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1496
+query: |
+  let queryFrequency = 5m;
+  let eventCapture = "moved from tenant ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}) to tenant ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})";
+  AzureActivity
+  | where ingestion_time() > ago(queryFrequency)
+  | where OperationNameValue == "Microsoft.Subscription/updateTenant/action"
+  | where isnotempty(Properties_d)
+  | extend Summary = tostring(Properties_d.message)
+  | extend EventCapture = extract_all(eventCapture, Summary)
```

Comment:
There should be a where clause here for empty EventCapture. Otherwise results will have rows which are not applicable to this scenario as well.

Reply:
The `Microsoft.Subscription/updateTenant/action` action only occurs when someone moves a subscription from one tenant to another. The `EventCapture `was meant to extract useful entities from the log itself, but is somewhat fragile because it is based on the `Summary `message. 

If EventCapture is empty, the format of the Summary message has changed but it all results should still be applicable to the scenario, since the scenario is fully based on `OperationNameValue == "Microsoft.Subscription/updateTenant/action"`.